### PR TITLE
[4354] - Site Policy - Implement basic Site Policies

### PR DESCRIPTION
**Pull request by aachavarria**
Form Engine Site Policy;
Upload site policy WIP;

### CODE REVIEW

File: static-assets/components/cstudio-forms/data-sources/img-desktop-upload.js [hunk 1]
```diff
@@ -52,13 +52,14 @@ YAHOO.extend(CStudioForms.Datasources.ImgDesktopUpload, CStudioForms.CStudioForm
...
-          topWin.refreshNodes(topWin.currentTextNode, false, false, topWin.myTreeAssets, null, true);
-        }
+        // Using emitSystemEvent on fileUpload avoid to use this to refresh the pathNavigator
+        // var topWin = window.top.CStudioAuthoring.ContextualNav
+        //   ? window.top.CStudioAuthoring.ContextualNav.WcmRootFolder
+        //   : null;
+        //
+        // if (topWin && topWin.currentTextNode && topWin.myTreeAssets) {
+        //   topWin.refreshNodes(topWin.currentTextNode, false, false, topWin.myTreeAssets, null, true);
+        // }
```
Comment (rart) [inline static-assets/components/cstudio-forms/data-sources/img-desktop-upload.js]: It must work with old tree too.
After you've tested, you may simple remove the commented code.

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — COMMENTED** [review]
`Search.tsx` needs merging with the latest (conflicts)

File: static-assets/components/cstudio-forms/data-sources/img-desktop-upload.js (hunk 1 above)
Comment (aachavarria) [reply static-assets/components/cstudio-forms/data-sources/img-desktop-upload.js]: the old tree on 4.0 uses the emitSystemEvent messages propagation

**Review by rart — APPROVED** [review]

**Comment by rart** [comment]
Merged@https://github.com/craftercms/studio-ui/pull/2156

**Pull request closed without merging**